src/test/integration/suite/create-app.test.ts

```diff
@@ -0,0 +1,77 @@
+import { expect } from 'chai';
+import { EditorView, VSBrowser, WebDriver, Workbench } from 'vscode-extension-tester';
```

Comment:
<!--
  0 failure: 
  1 warning:  src/test/integrat...
  
  
  DangerID: danger-id-Danger;
  File: src/test/integration/suite/create-app.test.ts;
  Line: 2;
-->

- :warning: src/test/integration/suite/create-app.test.ts line 2 – 'Workbench' is defined but never used. (@typescript-eslint/no-unused-vars)